src/main/resources/application.conf

```diff
@@ -36,6 +36,12 @@ workflow-options {
 
   // AES-256 key to use to encrypt the values in `encrypted-fields`
   base64-encryption-key: "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="
+
+  // Directory where to write per workflow logs
+  workflow-log-dir: "cromwell-workflow-logs"
+
+  // When true, per workflow logs will be deleted after copying
+  workflow-log-temporary: true
```

Comment:
yeah, we all have our personal preferences there are arguments for both sides, but perhaps if there's a clear bias one way or another among the big-name server software all having file logging on or off by default, we'd better meet user's expectations if we did what they did.